components/Form/Select/__tests__/Select.test.tsx

```diff
@@ -59,7 +58,7 @@ describe('Select', () => {
       const validate = () => ({ [fieldName]: 'Required' });
 
       const { container, findByText } = render(
-        <Formik initialValues={{ [fieldName]: '' }} validate={validate}>
+        <Formik initialValues={{ [fieldName]: '' }} validate={validate} onSubmit={() => {}}>
```

Comment:
Having a hard time with this test (and the rest of the tests in the "interactions" suite. 

I am getting the idea that the Alert is not being rendered in the test, so there is nothing for findByText('Required') to find?